package.json

```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "start": "node ./examples/server.js",
-    "test": "gulp test"
+    "test": "./node_modules/.bin/gulp test"
```

Comment:
This change shouldn't be needed? `npm` places `node_modules/.bin` in `$PATH` when running `scripts`.